views/partials/_whats-new.njk

```diff
@@ -4,13 +4,10 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds-from-desktop">
           <h2 id="whats-new" class="govuk-heading-l">What’s new</h2>
-          <p class="govuk-body">9 January: We’ve opened our user survey and released GOV.UK Frontend 5.8.0.</p>
-          <h3 class="govuk-heading-m">User Survey 2025</h3>
-          <p class="govuk-body">Complete the <a href="https://surveys.publishing.service.gov.uk/s/GOVUKDesignSystemSurvey25/" class="govuk-link">GOV.UK Design System and GOV.UK Frontend Survey 2025</a> before it closes on 31 January.</p>
-          <p class="govuk-body">We welcome your feedback if you have used (or tried to use!) the GOV.UK Design System or GOV.UK Frontend, whatever your level of expertise and your role. Your responses are invaluable. They help us understand the barriers you face and shape the improvements we work on. Please complete the survey before it closes on 31 January. Thanks!</p>
-          <h3 class="govuk-heading-m">We’ve released GOV.UK Frontend v5.8.0</h3>
-          <p class="govuk-body">This release includes a new <code>ConfigurableComponent</code> class to help you build your own configurable components.</p>
-          <p class="govuk-body">Read the <a href="https://github.com/alphagov/govuk-frontend/releases/tag/v5.8.0" class="govuk-link">full release notes</a> to see what’s changed.</p>
+          <h3 class="govuk-heading-m">4 March 2025: We’ve released GOV.UK Frontend v5.9.0</h3>
```

Comment:
We previously haven't put this 'dateline' as a heading... it's probably not necessary for this one?

```suggestion
          <p class="govuk-body">4 March 2025: We’ve released GOV.UK Frontend v5.9.0</p>
```